web/mappings.py

```diff
@@ -0,0 +1,423 @@
+# arr1 = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]
+# arr2 = [2, 3, 4, 5, 6, 7, 8, 9, 'C', 'F', 'G', 'H', 'J', 'M', 'P', 'Q', 'R', 'V', 'W', 'X']
+# arr3 = []
+#
+# my_file = open('../words.txt', 'r')
+# lines = my_file.readlines()
+# for line in lines:
+#     arr3.append("'" + line.strip() + "'")
+#
+# i = 0
+# j = 0
+# k = 0
+#
+# while i < len(arr1):
+#     while j < len(arr2):
+#         print("'" + str(arr1[i]) + str(arr2[j]) + "'" + ": " + arr3[k] + ',')
+#         j += 1

```

Comment:
If you look at the keys, looks like you can take the numbers 0...19 and concatenate them with 2...9, C, F, ...W, X.
The 2...9, C, F, ...W, X can be stored in an array and basically have an index pointer loop through that array, and another to indicate the number 0...19.
Better way for making the keys and adding their respective values to the map.